Client/Frontend/Browser/FaviconManager.swift

```diff
@@ -80,6 +82,7 @@ class FaviconManager : TabHelper {
                         })
                     }
                 }
+            NSNotificationCenter.defaultCenter().postNotificationName(FaviconManager.FaviconDidLoad, object: nil)
```

Comment:
You might experience a race condition with this notification because technically the favicon hasn't been added to the database yet when this gets fired. On https://github.com/mozilla/firefox-ios/pull/1946/files#diff-1dccca00bfd714476c188b2b63c6e659R76, a `Deferred` is return which wraps the async write to the database. If you want to make sure that the favicon is in the DB before firing you'll need to add an `upon` or `>>> effect()` block to it.